index.src.html

```diff
@@ -2819,6 +2842,18 @@
     The {{VideoFrame/duration}} getter steps are to return
     {{VideoFrame/[[duration]]}}.
 
+: <dfn attribute for=VideoFrame>hasAlpha</dfn>
+:: Whether or not the {{VideoFrame}} has an alpha channel. If this is `false`,
+    the frame is considered to be opaque, and must be rendered as such.
+    For planar {{PixelFormat}}s, the last {{Plane}} (highest index) in
+    {{VideoFrame/planes}} is an alpha plane. For interleaved {{PixelFormat}}s,
+    the alpha channel is interleaved within each block of related samples,
+    according to that {{PixelFormat}}'s specifications.
+
+ISSUE: {{VideoFrame}}s with an interleaved {{PixelFormat}} should omit copying
```

Comment:
After further discussion, I think we can go ahead and solve this here and now as described above. "The values of the 'X' channel are undefined and must be ignored when rendering." Implementers are free (and probably will) just copy out the alpha data as is with this additional signal that it should be ignored.